gateway-outbound.Dockerfile

```diff
@@ -1,3 +1,3 @@
 FROM asecurityteam/serverfull-gateway
-COPY api.yaml .
+COPY api-outbound.yaml .
```

Comment:
+1 on the name change. inbound/outbound seems much more specific than external/internal

Reply:
Should I change api.yaml to api-inbound.yaml?

Reply:
I, personally, find it clearer and would appreciate it.

Reply:
Fixed